homeassistant/bootstrap.py

```diff
@@ -48,7 +48,7 @@ def async_setup_component(hass: core.HomeAssistant, domain: str,
                           config: Optional[Dict]=None) -> bool:
     """Setup a component and all its dependencies.
 
-    This method need to run in a executor.
+    This method must be run in the event loop.
```

Comment:
I prefer if we follow the convention of the Python docs when it comes to coroutines:

> This method is a coroutine.